tests/sentry/api/endpoints/test_data_export.py

```diff
@@ -0,0 +1,43 @@
+from __future__ import absolute_import
+
+import six
+
+from django.core.urlresolvers import reverse
+from django.utils import timezone
+
+from sentry.models import ExportedData
+from sentry.models.exporteddata import ExportStatus
+from sentry.testutils import APITestCase
+
+
+class DataExportTest(APITestCase):
+    endpoint = "sentry-api-0-organization-data-export"
+
+    TEST_DATE_ADDED = timezone.now()
+
+    def test_simple(self):
```

Comment:
Are there user roles that cannot use data exports, or feature flags that control access to this?

Reply:
Yep! The backend API routes are already in `master` but they are feature flagged with `orgnizations:data-export`, so no end-user can access them. The frontend routes aren't referenced by any links, and throw errors since the APIs they depend on respond with 404s.

I will probably make a PR into `getsentry` to add it to internal users once I get more parts working, but for now, the